src/index.ts

```diff
@@ -34,6 +35,7 @@ export {
   Divider,
   Icon,
   Menu,
+  Message,
```

Comment:
Do we really want to export this component, or we should keep it private and use it only with the hook?

Reply:
the component could be useful when the user is not using function components 🤷

Reply:
apart from this I'd not export the component, but only the hook

Reply:
Then I'll remove it from here.